### PR TITLE
Version changes for Ganga->LHCbDIRAC tests

### PR DESCRIPTION
**Pull request by fstagni**
In this PR:

- bumped DIRAC version(s) used in dockerfiles
- if a specific version of LHCbDIRAC is requested, such version will be setup (this is mostly for testing purposes)

### CODE REVIEW

File: ganga/GangaDirac/test/Dockerfile [hunk 1]
```diff
@@ -16,20 +16,20 @@ RUN virtualenv venv && \
 
 RUN mkdir dirac_ui &&\
     cd dirac_ui &&\
-    wget -np -O dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/rel-v6r19/Core/scripts/dirac-install.py &&\
+    wget -np -O dirac-install https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py &&\
```
Comment (mesmith75) [inline ganga/GangaDirac/test/Dockerfile:19]: This doesn't work due to:
https://github.com/DIRACGrid/DIRAC/issues/3906

Comment (fstagni) [reply ganga/GangaDirac/test/Dockerfile:19]: Just fixed that

Comment (egede) [reply ganga/GangaDirac/test/Dockerfile:19]: test please

**Comment by fstagni** [comment]
The test fails, IIUC it's http://146.179.248.31:8080/blue/organizations/jenkins/Ganga-PR%2Fganga/detail/PR-1385/3/tests
But honestly I need a hint to understand what's wrong...

**Comment by mesmith75** [comment]
You can't use `best`. Revert back to explicitly asking for the `gcc49` platform.

**Comment by fstagni** [comment]
I did. But... why can't you use "-c best"? This is actually what we advise users to do.

**Comment by mesmith75** [comment]
No idea - try it on lxplus. You get nothing back for LHCbDirac.

**Comment by fstagni** [comment]
Ah... I see, but that's true for the "-l" option only. Otherwise we can use best. I'll commit once more.

**Review by mesmith75 — APPROVED** [review]

**Comment by fstagni** [comment]
Hi... are you going to merge this PR for the next release? We would need this in place to complete the certification of LHCbDIRAC release

**Comment by mesmith75** [comment]
Ok. We can release a new version tomorrow.

**Pull request merged**